Decred Wallet/Features/TransactionDetails/TransactionDetailsViewController.swift

```diff
@@ -8,262 +8,294 @@ import UIKit
-            UIPasteboard.general.string = text
-            
-            let alertController = UIAlertController(title: "",
-                                                    message: LocalizedStrings.copied,
+            let alertController = UIAlertController(title: LocalizedStrings.howToCopy,
+                                                    message: "",
                                                     preferredStyle: UIAlertController.Style.alert)
-            
-            alertController.addAction(UIAlertAction(title: LocalizedStrings.ok,
+
+            let attributedStringStyles = [ AttributedStringStyle(tag: "blue",
+                                                                font: UIFont.systemFont(ofSize: 14),
+                                                                color: UIColor.appColors.lightBlue)
+                                            ]
+            let defaultAttributedStringStyle = AttributedStringStyle(font: UIFont.systemFont(ofSize: 14),
+                                                                     color: UIColor.appColors.deepGray)
+            let attrMsgString =  Utils.styleAttributedString(LocalizedStrings.tapOnBlueText,
+                                                                styles: attributedStringStyles,
+                                                                defaultStyle: defaultAttributedStringStyle)
+            alertController.setValue(attrMsgString, forKey: "attributedMessage")
+
+            alertController.addAction(UIAlertAction(title: LocalizedStrings.gotIt,
                                                     style: UIAlertAction.Style.default,
                                                     handler: nil))
-            
             self.present(alertController, animated: true, completion: nil)
         }
     }
-    
-    func openLink(urlString: String) {
-        if let url = URL(string: urlString) {
-            let viewController = SFSafariViewController(url: url)
-            viewController.delegate = self as SFSafariViewControllerDelegate
-            self.navigationController?.pushViewController(viewController, animated: true)
-        }
+
+    @IBAction func showOrHideDetails(_ sender: Any) {
+        self.transactionDetailsTable.isHidden = !self.transactionDetailsTable.isHidden
+        self.showOrHideDetailsBtn.setTitle(self.transactionDetailsTable.isHidden ? LocalizedStrings.showDetails : LocalizedStrings.hideDetails, for: .normal)
     }
 }
 
 extension TransactionDetailsViewController: UITableViewDataSource, UITableViewDelegate {
     func numberOfSections(in tableView: UITableView) -> Int {
-        return 3
+        return 4
     }
-    
+
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return section == 0 ? self.generalTxDetails.count : 1
     }
-    
+
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         return 0.5
```

Comment:
The section separator lines are not of even height, some are faint.
<img width="309" alt="Screenshot 2020-02-07 at 1 08 24 AM" src="https://user-images.githubusercontent.com/18400051/73989542-68064080-4946-11ea-8675-7367f0a69fd9.png">

My guess is that the ones that are clearer are because they host another table that also has a border set. If so, those borders should be unset and this height set to 1 for all borders here to be more clearly visible.